Script/index.js

```diff
@@ -0,0 +1,14 @@
+const hamburger = document.querySelector(".hmburger");
+const navMenu = document.querySelector(".nav__mobile");
```

Comment:
- [ ] I noticed that the classes you are selecting here do not exist in the HTML file, or at least do not exist in the HTML file for this branch. And for that reason, when I click on the hamburger menu, nothing works. Kindly work on having these classes and their contents to make the Mobile menu work as expected before your project can be ready for approval 👍 
- [ ] Also, the style of the hamburger menu you are having does not match the original design given. Please work on the styles to match your design with the template chosen and also consider the Welcome text (this is supposed to be your name as the logo) style to match the original one as well. 👍 


**Your Header Menu**
![image](https://user-images.githubusercontent.com/50941074/195068320-ee0e2c96-09b7-4b8b-8f78-09713f871398.png)


**The Original Design for Header and Hamburger Menu**
![image](https://user-images.githubusercontent.com/50941074/195068094-cc3d565f-c533-4eb0-925a-9e0fac2ee47f.png)